core/src/main/java/com/linecorp/armeria/client/DefaultClientFactory.java

```diff
@@ -171,4 +173,15 @@ public void close() {
     void doClose() {
         clientFactoriesToClose.forEach(ClientFactory::close);
     }
+
+    private URI normalizeUri(final URI uri, final Scheme scheme) {
```

Comment:
Could you remove final from the arguments? 😀

Reply:
Done. Sorry didn't know this convention :-)  
Would you be open, if I open another PR later to add this as a checkstyle check (to prevent this in future)?